src/components/BooksList.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
-
-const data = [
-  { title: 'one book', author: 'one author' },
-  { title: 'second book', author: 'second author' }];
+import NewBook from './NewBook';
 
 function BooksList() {
+  const data = useSelector((state) => state.books);
   return (
-    <>
-      {data.forEach((book) => {
-        <Book title={book.title} author={book.author} />;
-      })}
-    </>
+    <div>
+      <h2>Books list</h2>
+      <ul>
+        {data.value.map((book) => (
+          <Book
+            key={book.item_id}
```

Comment:
- [ ] Kindly fix this error in the browser. As far I can see, it seems an error from your `key` value which is equal to `item1`. You could use `uuid` to create a random `id` 💯 

![image](https://github.com/jlcarrascof/bookstore-cms/assets/93630700/bf9c88e1-7193-4229-80e0-f83d7e62be21)

Reply:
Ohhh boy, yes ... 🙈🙈 I'll work on it Mr @BregornOriginal Julio 🙂🫡👍

Reply:
By error I left in this code:

Book.propTypes = {
  book: PropTypes.shape({
    **item_id: PropTypes.string.isRequired,**
    title: PropTypes.string.isRequired,
    author: PropTypes.string.isRequired,
    category: PropTypes.string.isRequired,
  }).isRequired,
};

item_id: as number .... **Now is fixed and working. Adding a new record, with an unique ID and generated by UUID.**


![ERROR-FIXED](https://github.com/jlcarrascof/bookstore-cms/assets/86327867/80553cad-ffdc-4beb-befb-0f0937cf18f5)